Source/WTF/wtf/spi/darwin/XPCSPI.h

```diff
@@ -256,9 +258,20 @@ void xpc_release(xpc_object_t);
 
 WTF_EXTERN_C_END
 
-inline std::span<const uint8_t> xpc_dictionary_get_data_span(xpc_object_t xdict, const char* key)
+inline std::span<const uint8_t> xpc_dictionary_get_data_span(xpc_object_t xdict, ASCIILiteral key)
 {
     size_t dataSize { 0 };
-    auto* data = static_cast<const uint8_t*>(xpc_dictionary_get_data(xdict, key, &dataSize));
+    auto* data = static_cast<const uint8_t*>(xpc_dictionary_get_data(xdict, key.characters(), &dataSize)); // NOLINT
     return unsafeMakeSpan(data, dataSize);
 }
+
+// ASCIILiteral version of XPC_ERROR_KEY_DESCRIPTION.
+static constexpr auto xpcErrorDescriptionKey = "XPCErrorDescription"_s;
+
+inline String xpc_dictionary_get_wtfstring(xpc_object_t xdict, ASCIILiteral key)
+{
+    auto* cstring = xpc_dictionary_get_string(xdict, key.characters()); // NOLINT
+    if (!cstring)
+        return { };
+    return String::fromUTF8(cstring);
+}
```

Comment:
We could possibly implement this function in another header file, since it is not SPI, but that is not a blocker for landing this patch :)